SmartDeviceLink/SDLSoftButtonManager.h

```diff
@@ -35,6 +35,9 @@ typedef void(^SDLSoftButtonUpdateCompletionHandler)(NSError *__nullable error);
  */
 @property (copy, nonatomic) NSArray<SDLSoftButtonObject *> *softButtonObjects;
 
+/**
+ All transitions made in-between beginUpdates and this method will occur as one RPC update.
+ */
```

Comment:
Shouldn't this be amended to say that this does not include button transitions since each transition will send a separate `Show` request?